src/controllers/categories/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Get, Produces, Route, Security, SuccessResponse } from 'tsoa'
+import { inject, injectable, singleton } from 'tsyringe'
+
+import { Logger, type ILogger } from '../../logger.js'
+
+import HomepageTemplates from '../../views/homepage.js'
+import { HTML, HTMLController } from '../HTMLController.js'
+
+@singleton()
+@injectable()
+@Security('oauth2')
+@Route('/')
+@Produces('text/html')
+export class HomepageController extends HTMLController {
```

Comment:
just wonder why HomepageController is under `categories` folder?

Reply:
agreed I think this should just be in the controllers directory as `homepageController.ts` or similar